References: bsc#1239013

virt-install incorrectly assumes SEV-SNP guests require EFI firmware
via a pflash device. For SNP guests, the firmware must be provided
as a ROM device.

--- virt-manager-5.0.0/virtinst/domain/launch_security.py.orig	2025-03-06 10:02:31.600954861 -0700
+++ virt-manager-5.0.0/virtinst/domain/launch_security.py	2025-03-06 14:42:46.001355525 -0700
@@ -43,7 +43,7 @@ class DomainLaunchSecurity(XMLBuilder):
                 self.policy = "0x07"
 
     def _set_defaults_sev_snp(self, guest):
-        if not guest.os.is_q35() or not guest.is_uefi():
+        if not guest.os.is_q35():
             raise RuntimeError(_("SEV-SNP launch security requires a Q35 UEFI machine"))
 
     def set_defaults(self, guest):
